Source/Mosa.Utility.SourceCodeGenerator/Mosa.Utility.SourceCodeGenerator.csproj

```diff
@@ -1,5 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+	<PropertyGroup>
+		<OutputType>WinExe</OutputType>
```

Comment:
Should be Exe so it can be run on Linux as well as this isn't a platform dependant tool

Reply:
> Should be Exe so it can be run on Linux as well as this isn't a platform dependant tool

No need to, WinExe means it will generate a platform-specific stub which also works on macOS and Linux.